src/types/index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './AppIcon';
+export * from 'src/types/Icon';
```

Comment:
Shorten import

Reply:
Done, and shortened the imports for Icon file